lib/release_notes/changelog_file.rb

```diff
@@ -22,8 +22,9 @@ def prepend_to_existing(changelog)
     end
```

Comment:
You also call `github_file` with no repo on line 20. I that going to cause issues?